src/avio.jl

```diff
@@ -864,6 +864,8 @@ end
 
 Count the total number of frames in the video by seeking to start, skipping
 through each frame, and seeking back to the start.
+
+For a faster alternative, try `get_number_frames`.
```

Comment:
```suggestion
For a faster alternative that relies on video container metadata, try `get_number_frames`.
```

Reply:
Maybe the language could be softer.. I may be a bit burned by experiences before this PR of the disagreement between the two methods

Reply:
In my experience they mostly agree now that most of the issues with frame dropping have been resolved. If you have any files where they still disagree, I think it would be great to see what the cause of the disagreement is!

Reply:
I committed your suggestion.